### PR TITLE
Fix/#1615 regex group

### PR DESCRIPTION
**Pull request by vgenev**
regex was not working properly with single resource version

### CODE REVIEW

**Review by rmothilal — APPROVED** [review]
Happy

File: src/util/helpers.js [hunk 1]
```diff
@@ -48,9 +48,10 @@ const getVersionFromConfig = (resourceString) => {
  */
 const parseResourceVersions = (resourceString) => {
   if (!resourceString) return {}
-  const resourceFormatRegex = /(([A-Za-z])\w*)=([0-9]+).([0-9]+)([^;:|],*)/g
+  const resourceFormatRegex = /((([A-Za-z])+)=([0-9]+)([. 0-9]*)(,?))+/
+  const match = resourceString.match(resourceFormatRegex)
   const noSpResources = resourceString.replace(/\s/g, '')
-  if (!resourceFormatRegex.test(noSpResources)) {
+  if (!(match && resourceString === match[0])) {
```
Comment (mdebarros) [inline src/util/helpers.js:54]: Isn't "test()" slightly more effecient?

Comment (vgenev) [reply src/util/helpers.js:54]: Its not in that case.
it gives *true* if the part of string matches, but not the whole part...

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**